python-3.11.yaml

```diff
@@ -106,6 +116,8 @@ subpackages:
           # pyconfig.h is needed at runtime... ugh.
           mkdir -p "${{targets.destdir}}"/usr/include/python3.11
           mv "${{targets.subpkgdir}}"/usr/include/python3.11/pyconfig.h "${{targets.destdir}}"/usr/include/python3.11
+          mkdir -p "${{targets.subpkgdir}}"/usr/lib
+          mv ${{targets.destdir}}/usr/lib/libpython3.so "${{targets.subpkgdir}}"/usr/lib
```

Comment:
-dev packages must be co-installable too.

Please move it to the -default sub package for now.

Reply:
bah. fixing. thanks.